yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1579,9 +1581,40 @@ func checker(sqlInfoArr []sqlInfo, fpath string, objType string) {
 	checkDDL(sqlInfoArr, fpath, objType)
 	checkForeign(sqlInfoArr, fpath)
 	checkRemaining(sqlInfoArr, fpath)
+	if utils.GetEnvAsBool("REPORT_UNSUPPORTED_PLPGSQL_OBJECTS", true) {
+		checkPlPgSQLStmtsUsingParser(sqlInfoArr, fpath, objType)
+	}
 	checkStmtsUsingParser(sqlInfoArr, fpath, objType)
 }
 
+func checkPlPgSQLStmtsUsingParser(sqlInfoArr []sqlInfo, fpath string, objType string) {
+	for _, sqlInfoStmt := range sqlInfoArr {
+		parserIssueDetector := queryissue.NewParserIssueDetector()
+		issues, err := parserIssueDetector.GetIssues(sqlInfoStmt.formattedStmt)
+		if err != nil {
+			log.Infof("error in getting the issues-%s: %v", sqlInfoStmt.formattedStmt, err)
+			continue
+		}
+		for _, issueInstance := range issues {
+			issue := convertIssueInstanceToAnalyzeIssue(issueInstance, sqlInfoStmt, objType)
+			schemaAnalysisReport.Issues = append(schemaAnalysisReport.Issues, issue)
+		}
+	}
+
+}
+
+func convertIssueInstanceToAnalyzeIssue(issueInstance issue.IssueInstance, sqlInfoStmt sqlInfo, objType string) utils.Issue {
+	return utils.Issue{
+		ObjectType:   objType,
```

Comment:
> is that correct?
yes correct.

Initially I thought it might make more sense for the analyze-schema to modify the issue as per its requirement, but now when I think that analyze is sending the function query to `GetIssues`, it will expect that IssueInstance should have the proper type of Object and name I believe so for PLPGSQL case I think it makes sense to modify the type/name in queryissue layer.